JQuery.Autocomplete/Remotion.ThirdPartyDependency.JQuery.Autocomplete.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <IncludeBuildOutput>false</IncludeBuildOutput>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <None Include="..\_._">
+      <Pack>true</Pack>
+      <PackagePath>\</PackagePath>
+    </None>
+  </ItemGroup>
+
+  <PropertyGroup>
+    <Owner>RUBICON IT GmbH</Owner>
+    <Version>1.1-pre</Version>
+    <Title>JQuery Autocomplete</Title>
+    <Description>jQuery Plugin: Autocomplete</Description>
+    <Authors>Dylan Verheul, Dan G. Switzer, Anjesh Tuladhar, Jörn Zaefferer</Authors>
+    <Copyright>Copyright (c) 2007 Dylan Verheul, Dan G. Switzer, Anjesh Tuladhar, Jörn Zaefferer</Copyright>
+    <PackageLicenseExpression>MIT OR GPL-3.0</PackageLicenseExpression>
```

Comment:
Add Notice.txt with the relevant info from AutoComplete.UI.ts (or Notice.txt, simliar to DLR)